tests/kuery/test_eql2kql.py

```diff
@@ -14,6 +14,8 @@ def validate(self, kql_source, eql_source):
 
     def test_field_equals(self):
         self.validate("field:value", "field == 'value'")
+        self.validate("field:value-*", "field == 'value-*'")
+        self.validate("field:value?", "field == 'value?'")
```

Comment:
can you create a separate method for these tests. Then call it with `elasticsearch_syntax` and for the operator us `:`. Also, the KQL value in these tests are not quoted like the PR author mentioned was the issue